Global.props

```diff
@@ -109,14 +109,14 @@
       Update for every public release. 
     -->
     <SemanticVersionMajor>1</SemanticVersionMajor>
-    <SemanticVersionMinor>2</SemanticVersionMinor>
+    <SemanticVersionMinor>3</SemanticVersionMinor>
```

Comment:
I actually plan to change to 1.2.3 because we have only bug fixes and no new features.